packages/hopli/src/migrate_safe_module.rs

```diff
@@ -0,0 +1,161 @@
+use crate::{
+    identity_input::LocalIdentityArgs,
+    key_pair::read_identities,
+    password::PasswordArgs,
+    process::{set_process_path_env, child_process_call_foundry_migrate_safe_module},
+    utils::{Cmd, HelperErrors},
+};
+use clap::Parser;
+use core_crypto::keypairs::Keypair;
+use core_crypto::types::ToChecksum;
+use ethers::{
+    types::U256,
+    utils::parse_units, //, types::U256, utils::format_units, ParseUnits
+};
+use log::{log, Level};
+use std::{env, str::FromStr};
+use utils_types::primitives::Address;
+
+/// CLI arguments for `hopli migrate-safe-module`
+#[derive(Parser, Default, Debug)]
+pub struct MigrateSafeModuleArgs {
+    #[clap(help = "Network name. E.g. monte_rosa", long)]
+    network: String,
+
+    #[clap(flatten)]
+    local_identity: LocalIdentityArgs,
+
+    #[clap(flatten)]
+    password: PasswordArgs,
+
+    #[clap(
+        help = "Specify path pointing to the contracts root",
+        long,
+        short,
+        default_value = None
+    )]
+    contracts_root: Option<String>,
+
+    #[clap(
+        help = "Ethereum address of safe",
+        long,
+        short
+    )]
+    safe_address: String,
+
+    #[clap(
+        help = "Ethereum address of node management module",
+        long,
+        short
+    )]
+    module_address: String,
+
+    #[clap(
+        help = "Hopr amount in ether, e.g. 10",
+        long,
+        short = 't',
+        value_parser = clap::value_parser!(f64),
+        default_value_t = 2000.0
+    )]
+    hopr_amount: f64,
```

Comment:
I would consider not need to fund again, just do the migration

Reply:
Updated accordingly